modules/plugins/treesitter/ts-textobjects/textobjects.nix

```
@@ -0,0 +1,9 @@
+{lib, ...}: let
+  inherit (lib.options) mkEnableOption;
+  inherit (lib.nvim.types) mkPluginSetupOption;
+in {
+  options.vim.treesitter.textobjects = {
+    enable = mkEnableOption "Treesitter textobjects";
+    setupOpts = mkPluginSetupOption "treesitter-textobjects" {};
```

an example would be appreciated
```suggestion
    setupOpts = mkPluginSetupOption "treesitter-textobjects" {} // {
      example = {
        select = {
          enable = true,
          lookahead = true,
          keymaps = {
            ["af"] = "@function.outer",
          },
      };
    };
  };
```

would hopefully make it clear enough that the whole thing goes under `textobjects` in the generated code

---

Done, thanks!